api.md

```diff
@@ -170,6 +171,114 @@ SELECT add_dimension('conditions', 'device_id', number_partitions => 2);
+current database for use with distributed hypertables.
+
+Newly created distributed hypertables will be able to make use of this
+data node for distributing data across the cluster. Note that existing
+distributed hypertables will not use the newly added data node. For
+existing distributed hypertables to use added data nodes, you need to
+use [`attach_data_node()`](#attach_data_node).
+
+The function will attempt to bootstrap the data node by creating the
+database where the remote hypertables will later be created and also
+load the TimescaleDB extension on the data node.
+
+#### Errors
+
+* An error will be given if an attempt is made to run the function
+  inside a transaction.
+* An error will be given if the remote database already exists on the
+  data node and `if_not_exists` is not set.
+
+#### Required Arguments [](add_data_node-required-arguments)
+
+| Name        | Description            |
+| ----------- | -----------            |
+| `node_name` | Name of the data node. |
+
+##### Parameter Description
+
+* `node_name` - Name to use for the node locally.
+
+#### Optional Arguments [](add_data_node-optional-arguments)
+
+| Name                 | Description                                           |
+|----------------------|-------------------------------------------------------|
+| `host`               | Host name for the remote data node.                   |
+| `port`               | Port to use on the remote data node.                  |
+| `database`           | Database name where the remote hypertable is created. |
+| `local_user`         | User name on the local node.                          |
+| `remote_user`        | User name on the remote data node.                    |
+| `password`           | Password on the remote data node.                     |
+| `if_not_exists`      | Do not fail if the hyper table already exists         |
+| `bootstrap_database` | Database to bootstrap.                                |
+| `bootstrap_user`     | User to used for bootstrapping.                       |
+| `bootstrap_password` | Password for user use when bootstrapping.             |
+
+##### Parameter Description
+
+* `host` - the name of the host to use for the remote data node. This
+  defaults to `'localhost'`.
+
+* `post` - Port to use for the remote data node. This defaults to the
+  default PostgreSQL port.
+
+* `database` - Name of the database to use on the remote data
+  node. Default is to use the current database in use when calling the
+  function.
+
+* `local_user` - Name of the local user. Default is to use the current
+  user.
+
+* `remote_user` - Name of the user on the remote data node. Default is
+  to use the same user as `local_user`.
+
+* `password` - Password to use on the remote data node. Defaults is to
+  not use a password.
+
+* `bootstrap_database` - The database to use when bootstrapping the
+  server as described above. The default is to use `'postgres'`.
```

Comment:
Is it possible to provide hyperlink to the description above?

Reply:
Does not seem to be easy to add labels arbitrarily.